packages/cli/src/ExternalSecrets/externalSecretsHelper.ee.ts

```diff
@@ -1,9 +1,9 @@
 import { License } from '@/License';
-import config from '@/config';
+import { ExternalSecretsConfig } from '@n8n/config/src/configs/external-secrets';
```

Comment:
should we export `ExternalSecretsConfig` in `index.ts` to avoid using the full path here?

Reply:
Changed the import to `GlobalConfig` to keep it consistent with the rest.

Reply:
I'd like to remove `GlobalConfig` at some point, but this should be fine for now 👍🏽